hopsworks-common/src/main/java/io/hops/hopsworks/common/kafka/KafkaController.java

```diff
@@ -74,14 +75,18 @@
 import org.apache.kafka.common.TopicPartitionInfo;
 import org.apache.zookeeper.KeeperException;
 import org.elasticsearch.common.Strings;
+import org.elasticsearch.common.collect.Tuple;
```

Comment:
@kai-chi can we use some other library for a Tuple, like https://commons.apache.org/proper/commons-lang/javadocs/api-3.9/org/apache/commons/lang3/tuple/Pair.html ? Motivation is to avoid having a dependency on elasticsearch just for this